### PR TITLE
More clarification on AsyncStorage

### PR DESCRIPTION
**Pull request by mscharlock**
Adding clarification as flagged by both Madhav and RE: a customer on Stack-Overflow

### CODE REVIEW

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-runtime.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-updateAsyncStorage/docs/excel/custom-functions-runtime.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C24%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201810242238207538-515%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=DHFZQHXA14txjwQ7PuGUvArQGZiT6T1fRs0huGPzog8%3D&st=2018-10-24T22%3A34%3A36Z&se=2018-11-24T22%3A39%3A36Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/excel/custom-functions-runtime.md [hunk 1]
```diff
@@ -66,17 +66,18 @@ ws.onerror = (error) => {
 
 Within a custom function (or within any other part of an add-in), you can store and access data by using the `OfficeRuntime.AsyncStorage` object. `AsyncStorage` is a persistent, unencrypted, key-value storage system that provides an alternative to [localStorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage), which cannot be used within custom functions. An add-in can store up to 10 MB of data using `AsyncStorage`.
 
+`AsyncStorage` is intended as a shared storage solution, meaning multiple parts of an add-in are able to access the same data. For example, tokens for user authentication may be stored in `AsyncStorage` because it can be accessed by both a custom function and add-in UI elements like the task pane. Similarly, if two add-ins share the same domain, they are also permitted to share information back and forth through `AsyncStorage`. However, because `AsyncStorage` can be a shared location, it is important to realize that it is possible to override key-value pairs.
```
Comment (Rick-Kirkham) [inline docs/excel/custom-functions-runtime.md]: Your friendly neighborhood pedantic old man wants "like the task pane" changed to "such as a task pane".

Comment (Rick-Kirkham) [inline docs/excel/custom-functions-runtime.md]: We know from experience that "domain" is ambiguous to many developers. We need to spell out where different subdomains count as the "same" domain. E.g. Are "funny.contoso.com" and "sad.contoso.com" the same domain or different domains for purposes of AsyncStorage?

Comment (mscharlock) [reply docs/excel/custom-functions-runtime.md]: You got it ;)

Comment (mscharlock) [reply docs/excel/custom-functions-runtime.md]: Good point. I will check with Madhav. My sense is that those are different RE: AsyncStorage but I will double check just to make sure.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-runtime.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-updateAsyncStorage/docs/excel/custom-functions-runtime.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C24%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201810242246408029-515%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=5ncoWu5q6fSg2x%2Fhi7a0e5OQZMU6jQUhCneDSnmuvMk%3D&st=2018-10-24T22%3A42%3A59Z&se=2018-11-24T22%3A47%3A59Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/excel/custom-functions-runtime.md (hunk 1 above)
Comment (mscharlock) [reply docs/excel/custom-functions-runtime.md]: Checked - yes, diff subdomains are indeed different instances of AsyncStorage. :)

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-runtime.md](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-updateAsyncStorage/docs/excel/custom-functions-runtime.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C24%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201810242301428258-515%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=3YAjQjFu2w6VQuNi3DCUF4KjfLS2DndLLrJBn5AUY20%3D&st=2018-10-24T22%3A58%3A00Z&se=2018-11-24T23%3A03%3A00Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**